src/runtime/policy/tests/policy-test.ts

```diff
@@ -866,7 +866,7 @@ policy MyPolicy {
     // The expected max read type variable should have `A { foo {a} }` for writeSuperset.
     const expected = await createTypeVarForSchema(
       'A',
-      '', /* TODO(b/175169555): should be 'foo: inline Foo {a: Text}'*/
+      'foo: inline Foo {a: Text}',
```

Comment:
Can you also take care of this?

https://github.com/PolymerLabs/arcs/blob/986e1341ac7bbeda33456795931b200dd02aaade/src/runtime/policy/ingress-validation.ts#L305